internal/pkg/library/library.go

```diff
@@ -70,12 +71,43 @@ func DownloadImageNoProgress(ctx context.Context, c *client.Client, imagePath, a
 	return DownloadImage(ctx, c, imagePath, arch, libraryRef, nil)
 }
 
+func getImageTags(image map[string]string) []string {
+	var ret []string
+	for t, _ := range image {
+		ret = append(ret, t)
+	}
+
+	return ret
+}
+
 // SearchLibrary searches the library and outputs results to stdout
 func SearchLibrary(ctx context.Context, c *client.Client, value string) error {
 	if len(value) < 3 {
 		return fmt.Errorf("bad query '%s'. You must search for at least 3 characters", value)
 	}
 
+	// If the user is searched for a container uri (eg. library/default/alpine) then
+	// try to get the image infomation, if unsuccessful, then search as usual.
```

Comment:
The terminology is a bit mixed here. You are actually getting information about a container - including all the tagged images for that container.